readme.md

```diff
@@ -100,6 +102,34 @@ snapchat
 stock market
 ```
 
+## Tumblr
```

Comment:
Naming is a joke. Referring to all the SJW whining that has recently migrated to Twitter from Tumblr.
For instance: https://www.reddit.com/r/SJWstories/comments/3mc0eq/i_used_to_be_a_radical_genderfluid_tumblr_user/